finality-aleph/src/sync/substrate/verification/cache.rs

```diff
@@ -150,41 +169,58 @@ where
             return Err(CacheError::SessionInFuture(session_id, upper_bound));
         }
 
+        // Prune old session data
```

Comment:
Move into separate function as it used to be, then you don't need this comment.

Reply:
so be it